### DISCUSSION TITLE
AWS RDS

### DISCUSSION THREAD

**Discussion by vincent0426** [pgroonga/pgroonga#281] [Q&A] [answered]
Hi, I'm able to install and use pgroonga on my local computer. But can't redo such processes when connected to AWS RDS. Is it now able to use it on AWS RDS or how can I do so? Thank you very much!

**Comment by kou** [comment, marked as answer]
AWS RDS doesn't support PGroonga.
If you want managed PGroonga, Supabase is an option. See https://supabase.com/blog/launch-week-6-community-day#postgres-ecosystem for details.

**Reply by zoec0110** [reply]
Could you please inform me if there are any plans to include RDS support in the near future?
Thanks

**Reply by kou** [reply]
We can't control it. It's controlled by AWS.
